user_docs/en/userGuide.md

```diff
@@ -2516,6 +2516,11 @@ This option is only available if reporting of spelling errors is enabled in NVDA
 This option allows the user to control if key presses generated by applications such as on-screen keyboards and speech recognition software should be processed by NVDA.
 This option is on by default, though certain users may wish to turn this off, such as those typing Vietnamese with the UniKey typing software as it will  cause incorrect character input.
 
+##### Maximum delay between two key presses for a repeated key press {#MaxRepeatedKeyDelay}
+
+This option allows to control the maximum delay between two key presses to perform a double or triple key presses.
+Increasing this delay may be especially useful for people using sticky keys or having some physical disability.
```

Comment:
I think "delay" should be changed to "timeout", but that is only applicable if you accept my earlier idea. For now, this is just an English language clarification.

```suggestion
Increasing this delay may be especially useful for people using sticky keys, or who have a physical disability.
```